test/scripts/run_test.py

```diff
@@ -176,14 +179,20 @@ def recover_deps():
 
     args = parser.parse_args()
 
+    # clean environment before test.
+    recover_deps()
```

Comment:
Why do we need to recover_deps here at the beginning of the test?

Reply:
i think before test ,after test ,and error occurs ,need do clean